components/class_loader/debug_class_loader.rst

```diff
@@ -3,6 +3,7 @@ Debugging a Class Loader
 
 .. caution::
 
-    The ``DebugClassLoader`` from the ClassLoader component was deprecated
-    in Symfony 2.5 and removed in Symfony 3.0. Use the
-    :ref:`DebugClassLoader provided by the Debug component <component-debug-class-loader>`.
```

Comment:
Do we need this old deprecation?

Reply:
We may remove it ... but I think that keeping it (only in 3.4) is also OK.